db/seeds.rb

```diff
@@ -1,7 +1,16 @@
+# frozen_string_literal: true
+
 # This file should contain all the record creation needed to seed the database with its default values.
 # The data can then be loaded with the bin/rails db:seed command (or created alongside the database with db:setup).
 #
 # Examples:
 #
 #   movies = Movie.create([{ name: "Star Wars" }, { name: "Lord of the Rings" }])
 #   Character.create(name: "Luke", movie: movies.first)
+
+Greeting.create(message: 'Hello, world!')
```

Comment:
* As mentioned by the previous CR i still get errors trying to run the app following instructions on your readme file 
* But i was able to figure out the issue 
* The config for webpack and babel is not correct [Kindly check the lesson link and add the correct config](https://betterprogramming.pub/react-with-rails-2022-bd28e1fcd355#ad0a)
* Bellow are screenshots of my results as it finally worked and the requirement for babel config 👇 
![image](https://user-images.githubusercontent.com/52497246/228477383-f2314564-6e64-4cae-9598-e58d8ee88b09.png)
![image](https://user-images.githubusercontent.com/52497246/228477403-64b32c57-a82c-4208-b1e6-68146ba2e343.png)
![image](https://user-images.githubusercontent.com/52497246/228477447-099fe0fd-43da-4d72-879b-cb2f787cb396.png)